packages/terra-breakpoints/docs/README.md

```diff
@@ -0,0 +1,149 @@
+# Terra Breakpoints
+
+`terra-breakpoints` contains components and utilities related to Terra's supported responsive breakpoints.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-breakpoints`
+
+## Usage
+
+### Breakpoints 
+
+An object containing the named breakpoint values is the default export of the `terra-breakpoints` package. These named breakpoints are defined as minimum values.
+
+|Breakpoint Name|Minimum Value|
+|---|---|
+|`tiny`|0px|
+|`small`|544px|
+|`medium`|768px|
+|`large`|992px|
+|`huge`|1216px|
+|`enormous`|1440px|
+
+Helper functions are also provided to to help determine what breakpoint is active for a given value.
```

Comment:
nit: Change `Helper functions are also provided to to help` to `Utility functions are also provided to help`